main.go

```diff
@@ -36,7 +36,6 @@ func main() {
 
 	if err := run(ctx); err != nil {
 		log.Fatal(ctx, "fatal runtime error", err)
-		os.Exit(1)
```

Comment:
It would be better to change the log level to `Error` rather than remove the os.Exit I think here.

Reply:
This is now addressed.